TicketsDemo.Data/OptionsForCalculationPrice/TeaCoffeeBedParametrs.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketsDemo.Data.Entities;
+
+namespace TicketsDemo.Data.OptionsForCalculationPrice
+{
+    public struct TeaCoffeeBedParametrs
```

Comment:
Please rename into 'PriceCalculationParameters' also i wonder why 'ticket' is extracted as a parameter to ticket creation parameters, looks pity